chia/_tests/pools/test_pool_cmdline.py

```diff
@@ -808,7 +808,7 @@ async def test_plotnft_cli_inspect(
 
     assert (
         json_output["pool_wallet_info"]["current"]["owner_pubkey"]
-        == "0xb286bbf7a10fa058d2a2a758921377ef00bb7f8143e1bd40dd195ae918dbef42cfc481140f01b9eae13b430a0c8fe304"
+        == "0x880afd6f9e123005655376e389015877e60060b768592809d2c746325d256edeb0017e1b406cba0832aa983e5c4bbf54"
```

Comment:
I'm a little confused why if we didn't actually change the derivation, this test needed to change?

Reply:
The previous logic actually would result in an initial index of `2`.  With the simplification, this changed to (correctly) be an initial index of `1` and that means the hardcoded values in the test needed to change.